### ISSUE TITLE
CPPopUpButton: addItemWithTitle on (CPURLConnection)connection didReceiveData or prototype AJAX call

### ISSUE THREAD

**Issue by cws-zz** [cappuccino/cappuccino#147]
I have got a strange behavior. If I add items to a CPPopUpButton on the "onSuccess" function of a prototype ajax call, both label divs of the collapsed CPPopUpButton use the font color "black", so that it appears "bold" instead of "white/black" combination.

Here is the a code snippet:

```
- (void)loadDepartments 
{ 
    new Ajax.Request('loadDepartments.php',
      {
        method:'post',
        parameters: {selection: "all"},
        onSuccess: function(nom, json){
            for (var i = 0; i < json.length; i++) {
                [ddnAbteilung addItemWithTitle: json[i].description];   
            }
        },
        onFailure: function(){ [self showAJAXErrorAlert]; }
      });
}
```

The web-inspector of Safari 4 beta shows me following divs:

```
<div style="position: absolute; cursor: default; z-index: 200; font: normal normal bold 12px/normal Arial; text-overflow: ellipsis; white-space: nowrap; overflow-x: hidden; overflow-y: hidden; top: 5px; left: 0px; width: 163px; height: 15px; ">Some json response text</div>
<div style="position: absolute; cursor: default; z-index: 150; font: normal normal bold 12px/normal Arial; overflow-x: hidden; overflow-y: hidden; white-space: nowrap; text-overflow: ellipsis; top: 6px; left: 0px; width: 163px; height: 15px; ">Some json response text</div>
```

Normally it shoud look like this:

```
<div style="position: absolute; cursor: default; z-index: 200; font: normal normal bold 12px/normal Arial; text-overflow: ellipsis; white-space: nowrap; overflow-x: hidden; overflow-y: hidden; top: 5px; left: 0px; width: 163px; height: 15px; ">Some json response text</div>
<div style="position: absolute; cursor: default; z-index: 150; font: normal normal bold 12px/normal Arial; overflow-x: hidden; overflow-y: hidden; white-space: nowrap; text-overflow: ellipsis; color: rgb(240, 240, 240); top: 6px; left: 0px; width: 163px; height: 15px; ">Some json response text</div>
```

What do you think could be the problem?
